README.md

```diff
@@ -147,13 +137,23 @@ The breakpoints are configured in `./src/scss/settings/_settings.breakpoints.scs
 ## File Generators
 To quickly add new parts to your app like components modules etc. the syntax to you can use is
 ```sh
-yo emakinacee-react:gen type name module
+react g <scaffold> <name> [module]
 ```
+
+Scaffold  | Usage
+---       | ---
+[Component](https://github.com/emakina-cee-oss/react-cli/wiki) | `react g component AwesomeComponent`
+[Module](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g module AwesomeModule`
+[Service](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g service AwesomeService`
+[Action](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g action awesomeAction SomeModule`
+[Factory](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g factory awesomeFactory SomeModule`
+[Compute](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g compute awesomeCompute SomeModule`
+
 See the following examples for more detail.
 
 ### Component
 ```sh
-yo emakinacee-react:gen component ComponentName
+react g component ComponentName
```

Comment:
I used strictly lowercase names with dashes too, since I worked on a Windows machine, and it caused issues with the file system.

Since it is the standard, and you hopefully do not need to develop in Windows anymore (bash on Windows) I could live with the PascalCase. Windows users would have to watch out, when a branch name changes case for example.